src/lib/debloat-helper/Remove-ItemPropertyVerified.psm1

```diff
@@ -0,0 +1,68 @@
+Import-Module -DisableNameChecking "$PSScriptRoot\..\Title-Templates.psm1"
+
+function Remove-ItemPropertyVerified() {
+    param (
+        [Parameter(Position = 0, Mandatory, ValueFromPipeline, ValueFromPipelineByPropertyName)]
+        [String[]] $Path,
+        [Parameter(Position = 1, Mandatory, ValueFromPipeline, ValueFromPipelineByPropertyName)]
+        [String[]] $Name,
+        [Parameter(ValueFromPipeline, ValueFromPipelineByPropertyName)]
+        [String[]] $Include,
+        [Parameter(ValueFromPipeline, ValueFromPipelineByPropertyName)]
+        [String[]] $Exclude,
+        [Parameter(ValueFromPipeline, ValueFromPipelineByPropertyName)]
+        [Switch]   $Force
+    )
+
+    Begin {
+        $ScriptBlock = "Remove-ItemProperty"
+        $Script:TweakType = "Exp/Reg"
+    }
+
+    Process {
+        If ((Get-Item -Path "HKLM:\SOFTWARE\Policies\Microsoft\Windows\WindowsUpdate\AU\").Property -ccontains $Name) {
```

Comment:
I've forgotten to change to the variable after testing on terminal. Changing to `$Path` as a Hotfix.